extensions/objects/templates/filepath-editor.directive.ts

```diff
@@ -685,7 +697,7 @@ angular.module('oppia').directive('filepathEditor', [
         };
 
         ctrl.discardUploadedFile = function() {
-          ctrl.resetFilePathEditor();
+          ctrl.resetFilePathEditor(false);
```

Comment:
Hm. Why false?

If it's because it's dependent on whether images are saved in local storage then why not just depend on ContextService.areImagesSavedInLocalStorage()?

Reply:
Removed it, it was because the same function was called when initializing the directive and deleting the image, but the lines in the function are duplicated in the init function anyway, so removed the call to this function from the init function.